commands/trade.js

```diff
@@ -114,11 +114,10 @@ module.exports = function container (get, set, clear) {
               limit: 1000
             }
             if (db_cursor) {
-              trade_cursor = db_cursor
```

Comment:
This line isn't actually necessary, was added in a previous PR I sent. This conditional branch once getNext is called following any valid data. The function passed to engine.update which set the db_cursor will also properly set the trade_cursor using a call to s.exchange.getCursor as one would expect.